src/main/web_src/src/components/ui/form/fieldArray/DollyFieldArray.js

```diff
@@ -8,6 +8,13 @@ import ExpandableBlokk from './ExpandableBlokk'
 import './dollyFieldArray.less'
 import { ErrorBoundary } from '~/components/ui/appError/ErrorBoundary'
 
+const numberColor = {
```

Comment:
Isede for å gi disse navn som tilsvarer fargen, noe som betyr at du må endre variablen hvis fargen endres. Kunne du gitt dem navn i forhold til bruksområde? 

```javascript
	ARRAY_LEVEL_ONE: '#CCE3ED',
	ARRAY_LEVEL_TOW: '#FFE5C2'
}
```